server/src/main/scala/com/lbs/server/lang/Pl.scala

```diff
@@ -379,4 +379,17 @@ object Pl extends Lang {
   override def canNotDetectPayer(error: String): String = s"Nie udało się ustalić płatnika. Powód: $error"
 
   override def pleaseChoosePayer: String = "<b>➡</b> Nie udało się ustalić domyślnego płatnika, wybierz jakiegoś."
+
+  override def youHaveAppointmentAt(reminder: Reminder): String =
+    s"""👍 Macie wizytę o ⏱ <b>${formatTime(reminder.appointmentTime.toLocalTime)}</b>!
+       |
+       |${capitalize(doctor)}: ${reminder.doctorName}
```

Comment:
externalize common parts?

Reply:
yeah, i agree. but have no time to do this now. but it's definitely a good idea.